tasks/5_profile-lookup/profile-lookup.js

```diff
@@ -0,0 +1,33 @@
+function lookupProfile(firstName, prop) {
+  let names = CONTACTS.map((c) => c.firstName);
+  if (names.includes(firstName)) {
+    let foundContact = CONTACTS.find((c) => c.firstName === firstName);
+    let props = foundContact[prop];
+    return props === undefined ? 'No such property' : props;
```

Comment:
What if `prop = undefined`? For example:
```js
{
  firstName: 'Akira',
  address: undefined,
}
```
[in operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in)